lib/views/home_content/map/post_map.dart

```diff
@@ -15,13 +17,28 @@ class PostMap extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final currentPosition = ref.watch(liveLocationServiceProvider);
+    final positionNotifier = ref.watch(mapProvider.notifier);
+
+    currentPosition.when(
+      data: (data) {
+        debugPrint("Live location: ${data!.latitude}, ${data.longitude}");
+        positionNotifier.animateCam(LatLng(data.latitude, data.longitude));
+        positionNotifier.redrawCircle(LatLng(data.latitude, data.longitude));
+      },
+      error: (error, _) {
+        throw Exception("Live location error: $error");
+      },
+      loading: () => (),
+    );
```

Comment:
I suggest we put this in the view-model, as this update is not triggered by interaction with the view.

Reply:
Done with e715c44